spf.gemspec

```diff
@@ -38,35 +39,31 @@ Gem::Specification.new do |s|
   ]
   s.homepage = "https://github.com/agaridata/spf-ruby"
   s.licenses = ["none (all rights reserved)"]
-  s.require_paths = ["lib"]
-  s.rubygems_version = "1.8.23.2"
+  s.rubygems_version = "2.4.6"
```

Comment:
This is a pretty radical requirement. Can we possibly do with an earlier version of rubygems? Maybe 2.0.0?